Formula/xmrig.rb

```diff
@@ -11,16 +11,23 @@ class Xmrig < Formula
     sha256 "3a274f5c98cd7fb6946e208bf2b4e1a5309ce43f92e937208e188324bef3a937" => :el_capitan
   end
 
+  option "without-httpd", "Build without built in http server and API"
+  option "without-aeon", "Disable CryptoNight-Lite support"
```

Comment:
I'm not sure I see the value of the `without-aeon` option: it does not reduce the list of dependencies, so it's kind of a freebie to keep it in

Reply:
That just replicate the additional cmake options provided in upstream wiki. Personally I don't use it neither.

https://github.com/xmrig/xmrig/wiki/OS-X-Build